docs/user/rstudio/ide/guide/ui/files.qmd

```diff
@@ -2,9 +2,7 @@
 title: "Managing Files"
 ---
 
-Created from <https://support.rstudio.com/hc/en-us/articles/200484448-Editing-and-Executing-Code-in-the-RStudio-IDE>
-
-RStudio supports syntax highlighting and other specialized code-editing features for:
+RStudio supports syntax highlighting and other specialized code-editing features for specific file types:
 
 -   R, Python, and SQL scripts
```

Comment:
We shouldn't have a "bullet list" with one entry. 
If you want it in this format, I would leave it inline with line five:

```suggestion
RStudio supports syntax highlighting and other specialized code-editing features for specific file types: R, Python, and SQL scripts
```

If you would like to break it out, I would have each file type as its own "line"
- R
- Python
- SQL scripts

Reply:
Perhaps this, as I think the goal is to "bucket" them and provide some interesting examples, where splitting into individual list items would be too long.

```
- Scripts in R, Python, SQL
- Computational and scientific documents in Quarto, RMarkdown, and TeX
- Web content such as HTML and CSS
- and many more
```

Reply:
Resolved in latest commit.